core/src/test/java/io/parsingdata/metal/EqualityTest.java

```diff
@@ -21,6 +21,7 @@
 import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertTrue;
 
+import static io.parsingdata.metal.AutoEqualityTest.DUMMY_STREAM;
```

Comment:
Move `DUMMY_STREAM` elsewhere? Seems a bit out of place in `AutoEqualityTest`.

Reply:
I agree, but moving it to `EqualityTest` doesn't really make sense. There's no `ByteStreamTest` or similar that makes sense as a location.